whois-endtoend/src/test/java/net/ripe/db/whois/changedphase3/util/RestRunner.java

```diff
@@ -41,16 +41,17 @@ public void create(final Scenario scenario) {
 
             logEvent("Creating", objectForScenario);
 
-            WhoisResources whoisResources = RestTest.target(context.getRestPort(),
```

Comment:
"whoisResources" is clearer (or use the full word rather than "Resp", i.e. "response")